engine/render/renderer/Renderer.h

```diff
@@ -76,20 +74,33 @@ class Renderer
 
     void ClearDeviceQueue()
     {
-        vkDeviceWaitIdle(device.Device());
+        vkDeviceWaitIdle(Context().GetVkLogicalDevice());
     }
 
     void SetClearValue(float r, float g, float b, float a)
     {
         clearValue = {{r, g, b, a}};
     }
 
+    static Vulkan::Context& Context()
+    {
+        return Get()->context;
+    }
+
+    static Renderer* Get()
+    {
+        return instance;
+    }
+
 private:
 
-    static VulkanDevice* deviceInstance;
+    static inline Renderer* instance {nullptr};
```

Comment:
inlining a static variable just makes it that the static variable doesn't need to be declared in the `.cpp` file. I've reverted this back to be declared in the `.cpp` since it might have some unexpected behaviour.